src/Api/user.ts

```diff
@@ -59,4 +60,19 @@ const getUserId = async () => {
   }
 };
 
-export { checkAuthUser, getUserInformation, getUserId, getAxiosHeader };
+const getUserNameList = async () => {
+  try {
+    const response = await api.get(`/users/get-users`);
```

Comment:
```suggestion
    const {data} = await api.get(`/users/get-users`);
```
이렇게 한번 벗겨내면 코드를 조금이라도 줄일 수 있을 것 같습니다!

Reply:
오오 그렇네요 👍 반영하겠습니다

Reply:
오오 그렇네요 👍 반영하겠습니다